src/vault.rs

```diff
@@ -1,5 +1,8 @@
 use std::{fmt, str::FromStr};
 
+use std::fs::File;
+use std::io::Read;
```

Comment:
You can combine these into a single statement like `use std::{io::Read, fmt, fs::File, str::FromStr};`